packages/@react-aria/virtualizer/src/Virtualizer.tsx

```diff
@@ -83,6 +83,14 @@ function Virtualizer<T extends object, V>(props: VirtualizerProps<T, V>, ref: Re
     }
   }, [isLoading, onLoadMore, state]);
 
+  useLayoutEffect(() => {
+    if (!isLoading && onLoadMore && !state.isAnimating) {
+      if (ref.current?.offsetHeight >= state.contentSize.height) {
```

Comment:
I'd prefer not to read from the DOM if we can. I think you should be able to do:

```suggestion
      if (state.contentSize.height <= state.virtualizer.visibleRect.height) {
```

(For readability I also flipped the condition)